src/AST/Class.cs

```diff
@@ -102,6 +102,9 @@ public class Class : DeclarationContext
         // True if the class represents a static class.
         public bool IsStatic;
 
+        // True if declared with 'struct' keyword
+        public bool DeclaredStruct;
```

Comment:
Wouldn't it seem erroneous to see some code like:
```C#
@class.IsStruct = CheckClassIsStructible(@class, Driver);
```
while the function itself uses the same property as:
```C#
if(!@class.IsStruct)
    return false;
```